docs/_templates/components_requirements.tmpl

```diff
@@ -0,0 +1,29 @@
+.. -*- mode: rst -*-
+
+Inmanta core and extensions versions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
+{% if data['component_requirements']%}
+.. list-table:: Inmanta core and extensions versions
+   :widths: 25 25
+   :header-rows: 1
+
+   * - Component
+     - Required version
+
+   {% for key, value in data["component_requirements"] | dictsort %}
+   * - {{ key|e }}
+     - {{ value|e }}
+   {% endfor %}
+
+.. note::
+
+    This information is also available as a constraints file in `requirements.components.txt <./requirements.components.txt>`_.
+    and in a machine-consumable format in the `compatibility.json <./compatibility.json>`_ file.
```

Comment:
A requirements file is also a machine-consumable format. But just for a specific tool, namely pip. 
```suggestion
    and in json format in the `compatibility.json <./compatibility.json>`_ file.
```